vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,157 @@
+
+```json
+{
+    "default-registry": {
+        "kind": "git",
+        "baseline": "b98e11a16474488633d8a1ab3e160da1f94b98e2",
+        "repository": "https://github.com/microsoft/vcpkg"
+},
+    "registries": [
+        {
+        "kind": "artifact",
+        "location": "https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",
+        "name": "microsoft"
+        }
+    ]
+}
+```
+
+There's a lot going on here, but lets break it down line-by-line starting with the `default-registry` JSON object:
+
+As the name suggests, the `"default-registry"` specifies the default location vcpkg should use to look for packages. The `"default-registry":{` key specifies the default package-registry settings containing details about the registry.
+
+* `"kind": "git",`: Specifies the type of registry. In this case, it's a Git repository.
+* `"baseline": "b98e11a16474488633d8a1ab3e160da1f94b98e2",`: Represents a set of versions which are likely to work together.
+* "`"repository": "https://github.com/microsoft/vcpkg"`: Is the URL of the Git repository to be used as the default registry.
+
+This tutorial only uses the default registry, but know that you can specify additional registries by adding them to the `registries` array. Note that different kinds of registries will have different attributes. In this case, the `registries` array contains one additional registry that's different from the default. Let's break it down:
+
+* `"kind": "artifact,"`: Specifies the type of this additional registry. In this example, it's an "artifact" type.
+* `"location": https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",`: The URL from which this artifact registry can be fetched.
+* `"name": "microsoft"`: Specifies the name for this additional registry, which is "microsoft".
+
+To learn more about `vcpkg-configuration.json` check out our reference [documentation](..\reference\vcpkg-configuration-json.md).
+
+Now that the project is properly set up, lets add the `fmt` library as a dependency and generate the project files.
+
+## Adding dependencies and project files
+
+Since this project will use the `fmt` library to print "Hello World", you'll want to add `fmt` to the "dependencies" list in the manifest. To do this, run:
+
+```bash
+..\vcpkg\vcpkg add port fmt
+```
+
+Your `vcpkg.json` should look like this:
+
+```json
+{
+    "name": "helloworld",
+    "version": "1.0",
+    "dependencies": [
+    "fmt"
+    ]
+}
+```
+
+Notice that your `vcpkg.json` file now lists `fmt` as a dependency. Now, create the `CMakeLists.txt` and `main.cpp` files:
+
+```bash
+touch CMakeLists.txt main.cpp
+```
+
+Fill in `CMakeLists.txt` with the necessary information to find and link the `fmt` package. The `CMakeLists.txt` should look like this:
+
+```cmake
+cmake_minimum_required(VERSION 3.10)
+
+project(HelloWorld)
+
+find_package(fmt CONFIG REQUIRED)
+
+add_executable(HelloWorld main.cpp)
+
+target_link_libraries(HelloWorld PRIVATE fmt::fmt)
+```
```

Comment:
Since we are likely to use the same code-snippets in the Windows and macOS tutorials we should put this code into its own file in a `snippets` subdirectory and use the `:::code :::` markdown extension to render it. There's an example of how to use snippets in the existing Getting Started with Manifest Mode article.